src/python/pants/pantsd/service/pants_service.py

```diff
@@ -234,7 +234,7 @@ def services(self) -> KeysView[PantsService]:
     def are_all_alive(self) -> bool:
         """Return true if all services threads are still alive, and false if any have died.
 
-        This method does not have sideeffects: if one service thread has died, the rest should be
+        This method does not have side-effects: if one service thread has died, the rest should be
```

Comment:
This is inconsistent with previous changes using "side effects".

Reply:
Wow, yeah that's inconsistent. I'm looking it up...